cfgov/paying_for_college/static/paying_for_college/disclosures/static/css/main.ie.css

```diff
@@ -0,0 +1 @@
r-top:none;border-bottom:none}.block__bg{background:#e7e8e9;padding:2.8125em 1.875em}.block__padded-top{padding-top:1.875em;margin-top:1.875em}.block__padded-bottom{padding-bottom:1.875em;margin-bottom:1.875em}.block__sub{margin-top:1.875em;margin-bottom:1.875em}.content-l_col.block,.content-l_col.block__sub{margin-top:0}.content__bleedbar .content_main:after{content:none}.content__bleedbar .content_sidebar{padding:2.8125em 0 .9375em 1.875em;margin-left:0;position:relative;z-index:1;background:0 0}.content__bleedbar .content_wrapper{position:relative}.content__bleedbar .content_wrapper:after{content:'';display:block;width:9999px;border-left:1px solid #b4b5b6;height:100%;position:absolute;top:0;z-index:0;margin-left:10px;background:#e7e8e9}.grid_column__left-divider:before,.grid_column__top-divider:before{display:block;background-color:#b4b5b6;content:''}.m-hero__overlay .m-hero_wrapper,.m-hero_image{background-position:center;background-repeat:no-repeat}.content__bleedbar.content__2-1 .content_wrapper:after{left:66.66666667%}.content__bleedbar.content__3-1 .content_wrapper:after{left:75%}.content_wrapper,.wrapper{max-width:1200px;padding-right:15px;padding-left:15px;margin:0 auto;clear:both}.content_wrapper__match-content,.wrapper__match-content{padding-left:30px;padding-right:30px;max-width:1200px}.lt-ie9 .content_wrapper,.lt-ie9 .wrapper{max-width:960px}.lt-ie9 body{min-width:800px}.grid_column__top-divider{margin-top:3.75em;border-left-width:15px}.grid_column__top-divider:before{height:1px;width:100%;margin-bottom:1.875em;margin-left:auto!important;position:static}.grid_column__left-divider{border-left-width:30px}.grid_column__left-divider:before{width:1px;margin-left:-30px;position:absolute;top:0;bottom:0}.m-hero__bleeding .m-hero_wrapper,.m-hero_image{position:relative}.m-hero{background-color:#e7e8e9}.m-hero_wrapper{display:table;padding-top:2.8125em;padding-bottom:2.8125em;min-height:195px}.m-hero_image-wrapper,.m-hero_text{margin-right:-.25em;display:table-cell;vertical-align:middle;box-sizing:border-box}.m-hero_text{border:solid transparent;border-width:0 15px;width:58.33333333%}.m-hero_heading{margin-bottom:.44117647em;font-size:2.125em;font-weight:400;letter-spacing:initial;line-height:1.25;text-transform:initial}.m-hero_subhead{font-size:22px;line-height:1.25}.m-hero_image-wrapper{border:solid transparent;border-width:0 15px;width:41.66666667%}.m-hero_image{padding-bottom:56.25%;height:0;background-size:contain}.m-hero__knockout{background-color:#5a5d61;color:#fff}.m-hero__bleeding .m-hero_image-wrapper{position:absolute;height:100%;margin-top:-2.8125em;margin-bottom:-2.8125em}.m-hero__bleeding .m-hero_image{padding-bottom:0!important;height:100%;width:100%;position:absolute;background-size:cover}.m-hero__overlay .m-hero_wrapper{background-size:cover}.m-hero__overlay .m-hero_image{display:none}.o-featured-content-module{position:relative;min-height:320px;background-color:#e7e8e9}.o-featured-content-module:after{content:'';display:tabl
```

Comment:
FWIW when I diff these CSS files with the ones in college-costs I see some differences; I'll defer to a FEWD to review further.

Reply:
We're building these assets now rather than relying on a static copies.